File: nodejs/connect-database.js
const { Client } = require('cassandra-driver');

async function run() {
  const client = new Client({
    cloud: { secureConnectBundle: '../secure-connect-test.zip' },
    credentials: { username: 'datastax', password: 'datastax' }
  });

  await client.connect();

  // Execute a query
  const rs = await client.execute('SELECT * FROM system.local');
  console.log(`Your cluster returned ${rs.rowLength} row(s)`);
  await client.shutdown();
}

// Run the async function
run();

File: python/connect_database.py
from cassandra.cluster import Cluster
from cassandra.auth import PlainTextAuthProvider

cloud_config= {
        'secure_connect_bundle': '../secure-connect-test.zip'
}
auth_provider = PlainTextAuthProvider('datastax', 'datastax')
cluster = Cluster(cloud=cloud_config, auth_provider=auth_provider)
session = cluster.connect()

row = session.execute("select count(*) from system.local").one()
if row:
    print(row[0])
else:
    print("An error occurred.")
